src/fidesctl/cli/commands/core.py

```diff
@@ -131,3 +132,22 @@ def parse(ctx: click.Context, manifests_dir: str, verbose: bool = False) -> None
     taxonomy = _parse.parse(manifests_dir=manifests_dir)
     if verbose:
         pretty_echo(taxonomy.dict(), color="green")
+
+
+@click.command()
+@click.pass_context
+@manifests_dir_argument
+@with_analytics
+def sync(ctx: click.Context, manifests_dir: str) -> None:
+    """
+    Update local resource files by their fides_key to match their server versions.
+    """
+
+    config = ctx.obj["CONFIG"]
+    # Do this to validate the manifests since they won't get parsed during the sync process
+    _parse.parse(manifests_dir)
```

Comment:
Seeing as this is a "full-send" command, what do you think about adding a "this will overwrite any local files, press any key to continue" sort of step here in lieu of the --dry flag?

Reply:
i figured they can `git reset --hard` 🤷 but I guess since the git check isn't in here yet that would be advisable? or maybe the git check just needs to be a part of this